### ISSUE TITLE
Main menu should watch for terminal resizes

### ISSUE THREAD

**Issue by mafagafogigante** [walls-of-doom/walls-of-doom#7]
Currently, only after switching the selected option the menu will be redrawn and the sizes fixed.

Capturing LINES and COLS every 10 ms or so should be viable performance-wise.

---

Note that #4 makes this less relevant.

**Comment by mafagafogigante** [comment]
No longer relevant because we are now using a fixed window size.
